apps/cookbook/src/app/showcase/fonts-showcase/fonts-showcase.component.html

```diff
@@ -1,31 +1,39 @@
 <p>
-  The font sizes can be set by either using the font classes, e.g. <code>kirby-text-large</code>, or
-  by using the right tags like <code>h1, h2 & h3</code>
+  The font styles can be set by using the right tags like <code>h1, h2, h3 & p</code> or using one
+  of the supplied classes, eg. <code>kirby-text-large</code> or <code>kirby-text-small</code>.
 </p>
 <p>Font can also be set with the font-size function directly on your own classes, e.g:</p>
 <pre>
   .my-own-class {{ '{' }}
-    font-size: font-size("l");
+    font-size: font-size('l');
     ...
   {{ '}' }}
 </pre>
+<p>
+  It is recommended that the tags or classes are used, as they gives support for
```

Comment:
Consider also mentioning how it is handled on Android. I know it comes out the box due to the PR text - but if I only read the the cookbook page I would not know 🤖